org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessDialog.java

```diff
@@ -312,10 +312,11 @@ private void hideBusyMask()
 	{
 		if (mask != null && mask.getParent() != null) {
 			mask.detach();
-			StringBuilder script = new StringBuilder("(function(){let w=zk.Widget.$('#");
-			script.append(getParent().getUuid()).append("');w.busy=false;");
-			script.append("})()");
-			Clients.response(new AuScript(script.toString()));
+			if(getParent() != null) {
+				StringBuilder script = new StringBuilder("var w=zk.Widget.$('#");
```

Comment:
this merge with master is wrong (i.e it should be using the (function(){...})()).

Reply:
yes thank you, fixing in a moment..